chain/network/src/peer_manager/peer_store/tests.rs

```diff
@@ -33,6 +33,23 @@ fn make_config(
     Config {
         boot_nodes: boot_nodes.iter().cloned().collect(),
         blacklist,
+        peer_states_cache_size: 10,
+        connect_only_to_boot_nodes,
+        ban_window: time::Duration::seconds(1),
+        peer_expiration_duration: time::Duration::days(1000),
+    }
+}
+
+fn make_config_with_cache_size(
+    boot_nodes: &[PeerInfo],
+    blacklist: blacklist::Blacklist,
+    connect_only_to_boot_nodes: bool,
+    peer_states_cache_size: u32,
+) -> Config {
```

Comment:
Cache size is independent from the rest of the config. How about just overriding the default value in the particular tests?